test/mergeSortLoop.js

```diff
@@ -0,0 +1,33 @@
+import { expect } from 'chai'
+import mergeSort from '../src/mergeSort'
+
+describe.only('mergeSort()', function(){
```

Comment:
When defining your test suite, you should also consider edge cases: how should merge sort behave with an empty array?  how should merge sort behave with a non-array (a string, collection, null, undefined, etc.)